tests/sample-v2-addon/tests/integration/components/ui/page-test.ts

```diff
@@ -38,7 +39,7 @@ module('Integration | Component | ui/page', function (hooks) {
 
     assert
       .dom('[data-test-page-title]')
-      .hasClass(/^sample-v2-addon/, 'We see the local class name.')
+      .hasClass(getClassForUiPage('title'), 'We see the local class name.')
```

Comment:
Unfortunately, in consuming projects, there is a loss of information regarding the type of `localClass`. At the moment, we are unable to provide a spell check. (Of course, the assertion will correctly fail if there is a typo.)

<img width="800" alt="" src="https://github.com/ijlee2/embroider-css-modules/assets/16869656/155ed600-66cd-4189-be0c-f7d3e075a456">